tests/unit/test_client.py

```
@@ -17,341 +17,151 @@
 
+
 @pytest.fixture
 def core_client():
-    """Fixture for CoreClient instance."""
-    return CoreClient()
+    """Fixture for CoreHttpClient instance."""
+    return CoreHttpClient()
 
-@pytest.fixture
+
+@pytest.fixture  
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)